tempesta_fw/cache.c

```diff
@@ -158,6 +159,33 @@ static TfwStr g_crlf = { .ptr = S_CRLF, .len = SLEN(S_CRLF) };
 	}								\
 	for ( ; c != h_end; ++c, c = (c == u_end) ? h_start : c)
 
+static unsigned int tfw_cache_method_noncacheable = (1 << TFW_HTTP_METH_POST);
```

Comment:
Please don't use so long names, `tfw_cache_meth_nc` would be enough for the variable and the functions at the below.